### PR TITLE
Minor bug fixes

### PR DESCRIPTION
**Pull request by MarkMankins**

### CODE REVIEW

File: panda/plugins/osi_linux/osi_linux.cpp [hunk 1]
```diff
@@ -367,7 +367,9 @@ void on_get_libraries(CPUState *env, OsiProc *p, GArray **out) {
 	return;
 
 error0:
-	g_array_free(*out, true);  // safe even when *out == NULL
+	if((out != NULL) && (*out != NULL)) {
```
Comment (m000) [inline panda/plugins/osi_linux/osi_linux.cpp]: What is the use of the second check? I remember that the first check will hush some glib warnings that may be annoying depending on the use. But I'm not sure for the use of the second check.

Comment (m000) [inline panda/plugins/osi_linux/osi_linux.cpp]: The first check is redundant. If `out == NULL` you're not calling the function properly. It is vain to try to make the code *not* crash.

Also, can you check for other similar cases in the file and add the `*out != NULL` check?

Comment (MarkMankins) [reply panda/plugins/osi_linux/osi_linux.cpp]: You are correct.  I'll commit an updated fix to address both issues.  Thanks!

**Comment by MarkMankins** [comment]
Travis take two

**Comment by moyix** [comment]
Thanks! These look good.

**Pull request merged**